core/database/arango/__init__.py

```
@@ -0,0 +1,20 @@
+
+# Try to import Unix socket client if available
+try:
+    from .arango_unix_client import ArangoUnixClient
+except ImportError:
+    pass
+
+__all__ = [
+    'ArangoDBManager',
+    'retry_with_backoff',
+    'ArangoUnixClient',
+]
```

Ruff (0.12.2)

`__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)